integration_tests/tests/test_helper_get_models.sql

```diff
@@ -0,0 +1,14 @@
+-- depends_on: {{ ref('model_data_a') }}
+-- depends_on: {{ ref('model_struct') }}
+-- depends_on: {{ ref('model_without_import_ctes') }}
+-- depends_on: {{ ref('model_without_any_ctes') }}
+
+{% if execute %}
+{% set actual_list = codegen.get_models(prefix='model_')|sort %}
+{% endif %}
+
+{% set expected_list %}
+['model_data_a', 'model_struct', 'model_without_any_ctes', 'model_without_import_ctes']
+{% endset %}
+
+{{ assert_equal (actual_list | trim, expected_list | trim) }}
```

Comment:
```suggestion
{{ assert_equal (actual_list, expected_list) }}
```

The `| trim` filters are unnecessary here - you can't remove whitespace from a list. it's used in other code in these tests because they’re comparing the string values